### ISSUE TITLE
API limitation?

### ISSUE THREAD

**Issue by jagauthier** [moehmeni/syncedlyrics#36]
After writing a script to parse my music and download lyrics I've found that after several queries I'll start getting an exception like this:

  File "musixmatch.py", line 90, in get_lrc
    tracks = body["track_list"]

This seems to be related to some API limitation.  If I sit for a few minutes, (5-6) I can execute queries again.
Is this limitation documented, and is there some way to capture it?

**Comment by moehmeni** [comment]
Yeah similar to #35, I made a commit to handle non-200 status code to return None so we request next providers. It seems it is a captcha from what I see in API JSON:
```
{'message': {'header': {'status_code': 401, 'execute_time': 0.0026848316192627, 'hint': 'captcha'}, 'body': []}}
```
